turborepo-tests/helpers/setup_example_test.sh

```diff
@@ -35,9 +35,15 @@ if [ "$package_manager" == "pnpm" ] && [ "$example_path" == "non-monorepo" ]; th
   package_manager_command="pnpm install --ignore-workspace"
 fi
 
-# All examples implement these two tasks
-# and it's reasonable to assume that they will continue to do so
-turbo_command="turbo build lint --continue --output-logs=errors-only"
+# with-svelte is flaky when building and check types at the same time, because the build process of Svelte involves type generation
+# If the types are generating while the type checking happens, it can cause flakes.
```

Comment:
Does this indicate that we should have `"check-types": {"dependsOn": ["build"]}` for the Svelte example to make sure that type generation completes before we try to check them?